src/main/java/com/gmail/nossr50/util/skills/SkillUtils.java

```diff
@@ -264,7 +274,7 @@ private static boolean isLocalizedSkill(String skillName) {
         return false;
     }
 
-    protected static Material getRepairAndSalvageItem(ItemStack inHand) {
+    static Material getRepairAndSalvageItem(ItemStack inHand) {
```

Comment:
My preference is that you explicitly declare public, private or static

Reply:
The keword `protected` is generally redundant for static methods if the class cannot be instantiated.
`package-private` and `package-protected` are equivalent in such a scenario, hence why you could argue that the protected modifier would be redundant and causes confusion, since the principle of inheritance isn't applied here.

I can add `protected` if you want, but there are valid reasons for omitting it in cases like this to avoid confusion with the actual accessibility of the method.

Reply:
In my comment I wasn't recommending that it be made protected, I was saying that I prefer the scope be explicitly defined. I don't know why it was originally protected, most likely it was a mistake.

Reply:
Ah, got it. I reckon it was for internal purposes only or something like that.
Although I don't really see a reason not to expose this method.
I think this might as well be `public` then, you agree?